sys/net/gnrc/network_layer/ndp/gnrc_ndp.c

```diff
@@ -458,12 +459,18 @@ void gnrc_ndp_rtr_sol_handle(kernel_pid_t iface, gnrc_pktsnip_t *pkt,
                 /* or unicast, if source is known */
                 /* XXX: can't just use GNRC_NETAPI_MSG_TYPE_SND, since the next retransmission
                  * must also be set. */
-                gnrc_ipv6_nc_t *nc_entry = gnrc_ipv6_nc_get(iface, &ipv6->src);
+                nc_entry = gnrc_ipv6_nc_get(iface, &ipv6->src);
                 xtimer_set_msg(&nc_entry->rtr_adv_timer, delay, &nc_entry->rtr_adv_msg,
                                gnrc_ipv6_pid);
             }
 #endif
         }
+        nc_entry = gnrc_ipv6_nc_get(iface, &ipv6->src);
+        if (nc_entry != NULL) {
```

Comment:
Because above the entry is definitely created before in https://github.com/RIOT-OS/RIOT/pull/5139/files#diff-c668f2091e40b20cfc8d5c4b5aeca3e1R401. This needs not be the case here, because this line is not passed, when sending of RAs is deactivated for this interface.